fred2/sexp_tree.cpp

```diff
@@ -4915,11 +4923,14 @@ void sexp_tree::update_help(HTREEITEM h)
 	}
 
 	code = get_operator_const(tree_nodes[i].text);
-	str = help(code);
-	if (!str)
-		str = "No help available";
+	auto str = help(code);
+	if (!str) {
+		text.Format("No help available\r\n\r\n%s", nodeComment.c_str());
+	} else {
+		text.Format("%s\r\n\r\n%s", str, nodeComment.c_str());
+	}
 
-	help_box->SetWindowText(str);
+	help_box->SetWindowText(text);
```

Comment:
Since the function argument is now a `CString`, it should have a `(LPCSTR)` cast.  This is usually automatic, but may not work with all build configurations.